tests/test_notes.py

```diff
@@ -121,7 +121,7 @@ def setup_unreleased_entries(dirpath):
             CategoryChange.ADDED,
             CategoryChange.FIXED,
         ]
-        authors = ['jsmith', 'jdoe', 'jsmith', 'jdoe', 'jsmith']
+        authors = ['jsmith', 'jdoe', '\n- jsmith\n- jdoe', 'jdoe', 'jsmith']
```

Comment:
We need to check what happens when new authors are added to the file. These values doesn't give us much because they are already there.

Reply:
I added a third author (not mentioned in the examples previously) to the tests. Please let me know if this is not what you were expecting.
Thank you.